Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,71 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="Styles.css" </head> <body>
+
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</h2>
+
+
+    <p1>By Carl Zimmer </p1>
+
+    <p2>Nov. 30, 2018</p2>
+
+    <figure>
+        <img src="Images/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg" alt="octopus swimming under the sea">
+        <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchersare
+            uncertain how the trait evolved.</figcaption>
+        <img <figure>
+
+        <p> To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus
+            pulling two halves of a coconut shell together to hide inside. Later the animal stacks the shells together
+            like nesting bowls — and carts them away.</p>
+
+        <p>“It suggests the octopus is carrying these tools around because it has some understanding they may be useful
+            in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of
+            Cambridge in Britain.</p>
+
+        <p>But his amazement is mixed with puzzlement.</p>
+
+        <p>For decades, researchers have studied how certain animals evolved to be intelligent, among them apes,
+            elephants, dolphins and even some birds, such as crows and parrots.</p>
+
+        <p>But all the scientific theories fail when it comes to cephalopods, a group that includes octopuses, squid
+            and cuttlefish. Despite feats of creativity, they <a href="https://doi.org/10.1016/j.tree.2018.10.010"><cite>lack
+                    some hallmarks</cite></a> of intelligence seen in other species.
+
+            <p>“It’s an apparent paradox that’s been largely overlooked in the past,” said Mr. Amodio. He and five
+                other experts on animal intelligence explore this paradox in a paper published this month in the
+                journal Trends in Ecology and Evolution.</p>
+
+            <p>For scientists who study animal behavior, intelligence is not about acing a calculus test or taking a
+                car apart and putting it back together. Intelligence comprises sophisticated cognitive skills that help
+                an animal thrive.</p>
+            <figure>
+                <img scr="Images/Unknown.jpeg" alt="octopus with tentacles floating in the water">
+                <figcaption>The plural of octupus is "octopi". This here is a singular "octopus".</figcaption>
+                <figure>
```

Comment:
This is just missing the closing slash:
```suggestion
                </figure>
```